src/main.js

```diff
@@ -210,6 +215,7 @@ app.on('ready', function() {
       else {
         trayIcon.setImage(trayImages.normal);
         trayIcon.setToolTip(app.getName());
+        mainWindow.flashFrame(false);
```

Comment:
It seems that the blinking is disabled when all unread messages have been read. Is this your expected behavior?